bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -575,5 +575,5 @@ spec:
   maturity: beta
   provider:
     name: MongoDB, Inc
-  version: 1.8.1
+  version: 1.8.2
   replaces: mongodb-atlas-kubernetes.v1.8.1
```

Comment:
Seems this was the problem with Openshift in the previous release, here it should be 1.8.0.
Now it looks correct

Reply:
Yes, there was a problem with 1.8.1 version that I had to fix manually